IBPSA/Resources/src/BoundaryConditions/Validation/BESTEST/WeatherBESTEST.py

```diff
@@ -0,0 +1,815 @@
+#####################################################################
+# This script is used to validate weather reading and postprocessing
+# for tilted and oriented surfaces according to the BESTEST standard
+# at the end it will create a folder in the current working directory
+# called simulations and inside there can be the .mat files and the
+# .json files or just the .json files
+# Creates folders in "Temp" directory
+# Copies library from local-git-hub repository
+# executes simulations and prints results in current working directory
+# ettore.zanetti@polimi.it                                 2020-03-11
+#####################################################################
+import json
+import os
+import shutil
+import numpy as np 
+import copy
+from datetime import date
+# Make code Verbose
+CodeVerbose = True
+# check if it just implements post-process (from .mat files to Json files)
+POST_PROCESS_ONLY = False
+# Erase old .mat files
+CLEAN_MAT = True
+# Erse anything but the Json file results in the ResultJson folder and .matfiles
+DelEvr = False
+#Get IBPSA library from gitHub
+FROM_GIT_HUB = False
+# Modelica IBPSA Library working branch
+#BRANCH = 'issue1314_BESTEST_weather'
+BRANCH ='master'
+#Path to local library copy
+LIBPATH = "C:\\Users\\ettor\\Documents\\Repositories\\MODELICA\\TRY\\modelica-ibpsa\\IBPSA"
```

Comment:
I thought since it could be helpful to have a local pathing, instead of having it hardwired it refers to the relative path inside the repository folder using "pathlib"